packages/app/components/StatusBadge/index.tsx

```diff
@@ -0,0 +1,112 @@
+import { useTranslation } from "../../common-i18n/use-translation";
+import React from "react";
+import styled, { css } from "styled-components/native";
+import { Theme } from "../../provider/theme/theme.config";
+import { MATCH_TYPE } from "../SupportSection/types";
```

Comment:
you will skip **SupportSection** component, so just copy types in this file from component